API.yaml

```diff
@@ -1267,6 +1269,17 @@ paths:
                       type: array
                       items:
                         type: integer
+                    table_search_expressions:
+                      type: object
+                      properties:
+                        columdId:
```

Comment:
Is there another property missing with the actual search expression string?

Reply:
I believe this would define a map where the key is "columnId" and the value is an integer, e.g. { columnId: 1 }.

We are trying to define it as a map where the key is an integer and the value is a string, e.g. { 1: "regex" }.

I think this is not supported by OpenAPI, the keys can only be strings?

Reply:
I'll update it using additionalProperties for dictionaries as described here: [OpenAPI Dictionaries](https://swagger.io/docs/specification/data-models/dictionaries/).

I'll also clarify in the description what the keys and what the values are supposed to be